src/HcfLcmTest.java

```diff
@@ -0,0 +1,54 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+public class HcfLcmTest
+{
+	HcfLcm obj;
+	@Before
+	public void initializeClass()
+	{
+		obj=new HcfLcm();
+	}
+	
+	@Test
+	public void testHcf1()
```

Comment:
Test method name should describe which functionality you are testing inside it. In this case it should be like testHcf_twoEvenNo

Reply:
Resolved